static/app/views/issueList/groupSearchViewTabs/draggableTabBar.tsx

```diff
@@ -137,7 +137,6 @@ export function DraggableTabBar({
       })
       .filter(defined);
     setTabs(newTabs);
-    onReorder?.(newTabs);
```

Comment:
decoupling the reordering behavior that updates the tab state from the request that gets sent to save the events.